src/gui/components/Button.js

```diff
@@ -0,0 +1,50 @@
+import { Text, Graphics } from 'pixi.js'
+import Component from './Component'
+
+export const DEFAULT_OPTIONS = {
```

Comment:
Why do you export `DEFAULT_OPTIONS` in all components ?
How do you think about a static property (like react do) on the class ?